Defra.Cdp.Backend.Api/Services/Deployments/DeploymentsServiceV2.cs

```diff
@@ -296,8 +296,13 @@ public async Task<DeploymentFilters> GetDeploymentsFilters(CancellationToken ct)
             .Distinct(d => d.Status, FilterDefinition<DeploymentV2>.Empty, cancellationToken: ct)
             .ToListAsync(ct);
 
-        var userFilter =
-            new FilterDefinitionBuilder<DeploymentV2>().Where(d => d.User != null && d.User.DisplayName != "n/a");
+        var userFilter = new FilterDefinitionBuilder<DeploymentV2>().Where(d =>
+            d.User != null &&
+            !string.Equals(d.User.DisplayName, "n/a", StringComparison.CurrentCultureIgnoreCase) &&
+            !string.Equals(d.User.DisplayName, "admin", StringComparison.CurrentCultureIgnoreCase) &&
+            !string.Equals(d.User.DisplayName, "github workflow", StringComparison.CurrentCultureIgnoreCase)
+        );
+        
```

Comment:
Oh my dayz 🤮 . Is there a way to have a list of names to ignore? :-D

Reply:
yeah thats I'd do it in js land

Reply:
iirc `$nin` is the not in list mongo operator